azavea/temperate
# Add Concern fixture; Update WeatherEvents fixtures to reference concerns

## Overview

This PR adds a fixture for some sample data for the `Concern` model. It also points some `WeatherEvents` towards these concerns. Renames `weather.json` to `weatherevents.json`.

### Demo

```
In [5]: for concern in Concern.objects.all():
   ...:     print('\n', concern.indicator.name,
   ...:           '\n', concern.tagline_positive,
   ...:           '\n', concern.tagline_negative,
   ...:           '\n', concern.is_relative,
   ...:           '\n', concern.weatherevent_set.all(), '\n')      
   ...:           

 total_precipitation 
 wetter 
 dryer 
 False 
 <QuerySet [<WeatherEvent: Rain storms>]> 


 heat_wave_incidents 
 more heat wave incidents 
 less heat wave incidents 
 True 
 <QuerySet [<WeatherEvent: Heat waves>]> 


 average_high_temperature 
 hotter 
 cooler 
 False 
 <QuerySet []> 


 dry_spells 
 more dry spells 
 less dry spells 
 True 
 <QuerySet [<WeatherEvent: Drought>]> 


 diurnal_temperature_range 
 larger average difference between daily low and high temperature 
 smaller average difference between daily low and high temperature 
 False 
 <QuerySet []> 


 extreme_precipitation_events 
 more extreme precipitation events 
 less extreme precipitation events 
 True 
 <QuerySet [<WeatherEvent: Extreme precipitation>]> 
```

### Notes

Fixtures were run through `python -m json.tool` for easier readability. 

## Testing Instructions

 * `./scripts/update`
 * You can run the demo above in `shell_plus` to see the data.

Closes #228
